infrastructure/expected-fail/unhandled-rejection.html

```diff
@@ -4,5 +4,6 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script>
+setup({ single_test: true });
```

Comment:
I'd like this change to be reverted too. Another test that looks like this would be fine.